### PR TITLE
HL roles page finished

### PR DESCRIPTION
**Pull request by BrianLi009**

### CODE REVIEW

File: hl-roles/README.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+- The constant drive to learn and adapt to new technology and environment
+
+## Professional Team
+
+### Career Success Director
+
+##### Qualifications 
+- 1+ years of experience in applying and landing internships/PEY .
+- A deep understanding of  the technical interview process and what makes a strong technical resume.
+- Excellent interpersonal and highly developed communication skills (verbal and written) .
```
Comment (ArsalaBangash) [inline hl-roles/README.md]: Remove after fullstop here

File: hl-roles/README.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+- Leadership acumen and expereince.
+- Train associates to help support student career growth.
+
+
+#### Responsibilities
+- Hold office hours to help students with resume critiques, career advice, and technical interview questions.
+- Organize and run weekly technical interview workshops: Discuss a problem and facilitate group problem solving.
+- Collaborate with outreach director to curate an online list of career-related resources using Markdown and Vuepress (Ex [Here](https://docs.google.com/document/d/11zgd7rl9bimzh-2R4dDid7wtZui_dNIHCWQdUk23LVY/edit)).
```
Comment (ArsalaBangash) [inline hl-roles/README.md]: with the* outreach director

File: hl-roles/README.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+### Outreach Director
+
+#### Qualifications
+- 1+ years of experience in applying and landing internships/PEY .
+- A solid understanding of internship and PEY process.
```
Comment (ArsalaBangash) [inline hl-roles/README.md]: of the* internship and PEY process

File: hl-roles/README.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+- Define requirements needed in order to be granted access to space. 
+- Coordinate with Andrew Wang to grant/revoke student access.
+- Maintain a room & equipment (VR/3D Printing) usage scheduling system.
+- Write business cases, manage hacklab finances, and interface with Donna Coulson.
+- Interface with UTM Resources and other departments. 
+- Actively seek out collaborations with other departments outside of MCS.
+
+#### Qualifications
+- A passion and a clear vision for improving the Computer Science Student community.
+- Compassion for understanding the landscape of the Computer Science Student experience from all its different communities.
```
Comment (ArsalaBangash) [inline hl-roles/README.md]: Could be shortened to "An understanding of the computer science student experience from all of its communities.

File: hl-roles/README.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+- Maintain a room & equipment (VR/3D Printing) usage scheduling system.
+- Write business cases, manage hacklab finances, and interface with Donna Coulson.
+- Interface with UTM Resources and other departments. 
+- Actively seek out collaborations with other departments outside of MCS.
+
+#### Qualifications
+- A passion and a clear vision for improving the Computer Science Student community.
+- Compassion for understanding the landscape of the Computer Science Student experience from all its different communities. 
+- Excellent interpersonal and highly developed communication skills (verbal and written).
+- Detail-oriented: ability to handle multiple projects at once and prioritize effectively.
```
Comment (ArsalaBangash) [inline hl-roles/README.md]: Not sure if detail oriented is necessary here. Why not just go with The ability to handle ...

File: hl-roles/README.md [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+- Organize and run weekly technical interview workshops: Discuss a problem and facilitate group problem solving.
+- Collaborate with outreach director to curate an online list of career-related resources using Markdown and Vuepress (Ex [Here](https://docs.google.com/document/d/11zgd7rl9bimzh-2R4dDid7wtZui_dNIHCWQdUk23LVY/edit)).
+
+### Outreach Director
+
+#### Qualifications
+- 1+ years of experience in applying and landing internships/PEY .
+- A solid understanding of internship and PEY process.
+- Excellent interpersonal and communication  and skills (verbal and written) .
+- Detail-oriented: Able to handle multiple projects at once and prioritize effectively.
```
Comment (ArsalaBangash) [inline hl-roles/README.md]: Not sure if detail oriented is necessary here. Why not just go with The ability to handle ...

File: hl-roles/README.md [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+##### Qualifications 
+- 1+ years of experience in applying and landing internships/PEY .
+- A deep understanding of  the technical interview process and what makes a strong technical resume.
+- Excellent interpersonal and highly developed communication skills (verbal and written) .
+- Leadership acumen and expereince.
+- Train associates to help support student career growth.
+
+
+#### Responsibilities
+- Hold office hours to help students with resume critiques, career advice, and technical interview questions.
```
Comment (ArsalaBangash) [inline hl-roles/README.md:38]: In addition to Office hours, do we have bookable appointment slots?

**Review by ArsalaBangash — CHANGES_REQUESTED** [review]

**Review by BrianLi009 — COMMENTED** [review]
Updated Pull Request

**Review by ArsalaBangash — APPROVED** [review]
Great work!

**Pull request merged**